TicketsDemo/App_Start/NinjectWebCommon.cs

```diff
@@ -74,10 +75,9 @@ private static void RegisterServices(IKernel kernel)
             kernel.Bind<ITicketService>().To<TicketService>();
             kernel.Bind<IReservationService>().To<ReservationService>();
 
-            //todo factory
             kernel.Bind<IPriceCalculationStrategy>().To<DefaultPriceCalculationStrategy>();
```

Comment:
still using DefaultPriceCalculationStrategy, newly created ones are not registered and subsequently are not used